### PR TITLE
Coverage support

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,10 +111,56 @@ CARGO_FLAGS += --release
 TARGET_DIR=$(BINDIR)/target/release
 endif
 
+ifeq ($(COV),1)
```
Comment (chayim) [inline Makefile:114]: We should add comments as to why. There's no way anyone remembers this in the future.

File: Makefile [hunk 2]
```diff
@@ -209,6 +260,25 @@ pack:
 
 #----------------------------------------------------------------------------------------------
 
+COV_EXCLUDE_DIRS += deps tests
```
Comment (chayim) [inline Makefile:263]: Can these move into readies/mk/coverage* and maybe add a flag for USES_RUST or similar?

**Review by chayim — DISMISSED** [review]

**Review by oshadmi — APPROVED** [review]
👍🏼

**Comment by gkorland** [comment]
fix #406

**Pull request merged**